### PR TITLE
PR: ToBlockingCollection with capacity

### PR DESCRIPTION
**Pull request by rj2Skipper**
this PR fixes issue #3

### CODE REVIEW

**Comment by rj2Skipper** [comment]
reopen-branch
issue #3

File: ConcurrentPriorityQueue/Core/IPriorityQueue.cs [hunk 1]
```diff
@@ -5,6 +5,7 @@ namespace ConcurrentPriorityQueue.Core
 	public interface IPriorityQueue<T>
 	{
 		Result Enqueue(T item);
+        int Capacity { get; }
```
Comment (noamyogev84) [inline ConcurrentPriorityQueue/Core/IPriorityQueue.cs:8]: Please add tests

File: ConcurrentPriorityQueue/Core/ConcurrentPriorityQueue.cs [hunk 2]
```diff
@@ -16,6 +16,10 @@ public class ConcurrentPriorityQueue<T, TP> : IConcurrentPriorityQueue<T, TP>
 
         public int Count => _internalQueues.Count == 0 ? _internalQueues.Count : 
 	        _internalQueues.Values.Select(q => q.Count).Aggregate((a, b) => a + b);
+        public int Capacity
+        {
+            get { return _capacity; }
+        }
```
Comment (noamyogev84) [inline ConcurrentPriorityQueue/Core/ConcurrentPriorityQueue.cs:22]: modify to expresstion
`Capacity => _capacity`

**Comment by noamyogev84** [comment]
Hey @rj2Skipper what's up?
please make sure to take latest `develop` before you proceed with this...
Thanks :)

**Pull request closed without merging**